frontend/src/components/custom-tools/document-parser/DocumentParser.jsx

```diff
@@ -98,9 +107,14 @@ function DocumentParser({
     const body = {
       [`${name}`]: value,
     };
+
+    let url = `/api/v1/unstract/${sessionDetails?.orgId}/prompt-studio/prompt/${promptDetails?.prompt_id}/`;
```

Comment:
@tahierhussain Let's have a helper function to create the URL since we are using it in multiple places.

Reply:
@athul-rs Thanks for the suggestion. I have created a helper function (`promptUrl(urlPath)`) to return the URL that start with `/prompt-studio/prompt/`.

Reply:
:+1: